src/main/java/com/openlattice/chronicle/services/message/MessageService.kt

```diff
@@ -30,11 +33,68 @@ open class MessageService(
 ) : MessageManager {
 
     companion object {
-        protected val logger = LoggerFactory.getLogger(MessageService::class.java)
+        protected val logger: Logger = LoggerFactory.getLogger(MessageService::class.java)
+        private const val MAX_FAILURE_ATTEMPTS = 5
+
+    }
+
+    open fun getSearchTermString(propertyTypeId: UUID, searchString: String): String {
+        return "entity.$propertyTypeId:\"$searchString\""
+    }
+
+    @Throws(ExecutionException::class)
```

Comment:
Why did you add this? You don't need it, plus the try-catch already catches it.

(You can also remove it from `recordMessagesSent`)

Reply:
I'm so confused, I didn't add it. but it was there locally.

Reply:
I though it came from develop when I merged develop and resolved changes

Reply:
There's another `@Throws` you can remove on line 100.

Reply:
Kotlin conversion will often add these in.